client/src/redux/reducers/requestAccess.reducers.js

```diff
@@ -0,0 +1,41 @@
+export const initialRequestAccessState = {
+  creating: false,
+  creatingError: false,
+  creatingErrorMessage: '',
+  creatingSuccess: false,
+  createdTicketId: null
```

Comment:
I know this matches up with the state names for the `TicketCreateForm`, so this change request is not required. But here are some suggestions on state semantics to improve naming parity with other states
```suggestion
  loading: false,
  error: null,
  ticketId: null
```
And you can determine `success` state if `ticketId` has a value, and `error` is `null`. Similarly `errorMessage` could become `error`